packages/app/src/explorer/utils.ts

```diff
@@ -26,3 +26,28 @@ export function needsNxBadge(
 
   return false;
 }
+
+export function getActiveIndex(
+  buttonList: HTMLButtonElement[],
+  element: Element
+): number | undefined {
+  const { activeElement } = document;
+
+  if (
+    activeElement instanceof HTMLButtonElement &&
+    element.contains(activeElement)
+  ) {
+    return buttonList.lastIndexOf(activeElement);
+  }
+
+  if (activeElement === document.body || activeElement === undefined) {
+    const selectedElement = element.querySelector<HTMLButtonElement>(
```

Comment:
`selectedBtn`?

Reply:
Still not short enough to fit on one line :smile: